cclient/api/examples/example_get_transactions_to_approve.c

```diff
@@ -13,7 +13,7 @@
 void example_get_transactions_to_approve(iota_client_service_t *s) {
   printf("\n[%s]\n", __FUNCTION__);
   retcode_t ret = RC_ERROR;
-  flex_trit_t reference[FLEX_TRIT_SIZE_243];
+  // flex_trit_t reference[FLEX_TRIT_SIZE_243];
```

Comment:
Shouldn't this comment (and some others below) just be removed?

Reply:
hmm...let me add a macro on it.